python/lsst/pipe/tasks/selectImages.py

```diff
@@ -34,6 +34,37 @@
 from lsst.utils.timer import timeMethod
 
 
+def meetsVisitSummaryMinValues(visit, visitSummary, visitSummaryMinValues, logger):
```

Comment:
I'm not sure why this is a free function?

Reply:
Because I wanted to share it between `BestSeeingSelectVisitsTask` and  `BestSeeingQuantileSelectVisitsTask` and they currently don't have any relationship. That's kind of what I meant by "It was tempting to refactor it."  
`_meets` it is.